src/ci/docker/x86_64-gnu-tools/checkregression.py

```diff
@@ -4,6 +4,9 @@
 import sys
 import json
 
+# Regressions for these tools does not cause failure
+REGRESSION_OK = ["rustc-guide"]
```

Comment:
```suggestion
REGRESSION_OK = ["rustc-guide", "miri"]
```

Just to fix #62266 as well 🙃

Reply:
This does not seem right. I think we *do* want an error if the Miri submodule changes and that does not fix Miri, we just don't want the beta-cutoff-week thing.

Reply:
@RalfJung  This `checkregression.py … regressed` is only executed during the beta week. The submodule change checked is in `status_check "submodule_changed"`.

Sorry for the pasta 😂

Reply:
Oh I see. There should be comments here saying that. :)